ENH: Build ITK Video BridgeOpenCV as an external module

naucoin opened the pull request:
Point directly to the OpenCV bridge module in Slicer's ITK
source code directory and build it into the extension's build
directory. Relies on some new changes to ITK that will allow
it to build as an external module.

Add logic to the OpenCV module, use it to test registering the OpenCV IO factory,
linking properly to the ITK OpenCV bridge that is built as an external
module. Compile but don't run the bridge and OpenCV calls as they'll cause
a crash without proper settings.

fedorov commented:
@ebremer since I do not have write access to this repository, it will be your duty to review pull requests and merge them with the master. Let me know if you need help with this.

Commit:
ENH: build ITK Video BridgeOpenCV as an external module

Point directly to the OpenCV bridge module in Slicer's ITK
source code directory and build it into the extension's build
directory. Relies on some new changes to ITK that will allow
it to build as an external module.

Add logic to the OpenCV module, use it to test registering the OpenCV IO factory,
linking properly to the ITK OpenCV bridge that is built as an external
module. Compile but don't run the bridge and OpenCV calls as they'll cause
a crash without proper settings.
